Automattic/newspack-plugin
# Fix Plugins Screen Script/Stylesheet Paths

### All Submissions:






### Changes proposed in this Pull Request:

Alters how the plugins screen script and stylesheet are transpiled. Previously the stylesheet was Sass processed by the build script and the script was loaded untouched. Now the script is transpiled by Webpack, the stylesheet is imported into the script, and the resulting files are placed at the root of the `dist` directory like all other scripts and styles. 

### How to test the changes in this Pull Request:

1. `npm run clean && npm run build && npm run release:archive`
2. Unzip `assets/release/newspack-plugins.zip`. 
3. Verify it contains `dist/plugins-screen.css`, `dist/plugins-screen.rtl.css`, and  `dist/plugins-screen.js`.
4. Spin up a Jurassic Ninja instance. Install the Newspack Plugin from the zip.
5. Navigate to the Plugins screen (`/wp-admin/plugins.php`). 
6. Verify there are dashes before each Newspack managed plugin, and no checkbox next to installed ones.

<img width="1627" alt="Screen Shot 2020-02-25 at 3 32 12 PM" src="https://user-images.githubusercontent.com/1477002/75285131-066a2100-57e4-11ea-80b4-66bec5ad448b.png">

### Other information:
